aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -155,7 +155,10 @@ module aptos_framework::account {
     const ENO_SUCH_ROTATION_CAPABILITY_OFFER: u64 = 18;
     // The signer capability is not offered to any address
     const ENO_SIGNER_CAPABILITY_OFFERED: u64 = 19;
+    // This account has exceeded the allocated GUIDs it can create. It should be impossible to reach this number for real applications.
+    const EEXCEEDED_MAX_GUID_CREATION_NUM: u64 = 20;
```

Comment:
Why is that not covered by gas?

Reply:
Account and Objects can overlap since accounts can be implicitly created. Then someone could send arbitrary coins to the account to create valid event streams that would overlap with the Object event streams. Adding comments.